components/grid/columns/menu.md

```diff
@@ -130,58 +124,56 @@ You can organize the columns in the [Column Chooser](#column-chooser) in differe
     
     * If you set the `Title` parameter of the `GridColumnMenuChooserItem` it will override the value of the `Title` parameter of the corresponding Grid Column. 
 
-![columns organized in groups](images/column-menu-sections-example.png)
 
->caption Organize the columns in the Column Chooser in sections
+### Example
 
-````CSHTML
-@* Organize the columns in the Column Chooser in some sections. Use the Id and ColumnId parameters of the Grid Column and GridColumnMenuChooserItem respectively to relate them. *@
+The following example shows basic configuration of the `ColumnMenuSettings`.
 
-<TelerikGrid Data="@MyData"
+The columns in the Column Chooser are divided in sections. The Lockable option is disabled from the Column Menu. Filtering  is disabled from the Column Menu, so the Grid can use FilterRow. The `Id` column has no Column Menu and the `HireDate` column is not visible in Column Chooser.
```

Comment:
```suggestion
The columns in the Column Chooser are divided in sections. The Lockable option is disabled from the Column Menu. Filtering in the Column Menu is disabled, so the Grid can use a `FilterRow`. The `Id` column has no Column Menu and the `HireDate` column is not visible in Column Chooser.
```

Reply:
Use "divided into" instead of "divided in".